server/game/cards/07-GR/Soultender.js

```diff
@@ -0,0 +1,35 @@
+const Card = require('../../Card.js');
+
+class Soultender extends Card {
+    // Play:/After Reap: Move each A from each Specter creature to the common
+    // supply. Fully heal and ward each Specter creature.
+    setupCardAbilities(ability) {
+        this.play({
+            reap: true,
+            gameAction: ability.actions.removeAmber((context) => ({
+                all: true,
+                target: context.game.creaturesInPlay.filter((c) => c.hasTrait('specter'))
+            })),
+            then: {
+                alwaysTriggers: true,
+                gameAction: [
+                    ability.actions.heal((context) => ({
+                        target: context.game.creaturesInPlay.filter((c) => c.hasTrait('specter')),
+                        fully: 3
```

Comment:
well, it will work, but should be `fully: true`, right?

Reply:
Haha, sorry, yes, thanks for fixing.